lib/rules/no-unused-prop-types.js

```diff
@@ -722,6 +729,16 @@ module.exports = {
           }
           ignorePropsValidation = true;
           break;
+        case 'CallExpression':
+          if (
+            configuration.propWrapperFunctions &&
+            configuration.propWrapperFunctions.indexOf(propTypes.callee.name) !== -1 &&
```

Comment:
It might be worth converting the propWrapperFunctions to a structure with constant time lookup, like a Set, to keep this path fast.